<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeStepsManager_ui</class>
 <widget class="QGroupBox" name="TimeStepsManager_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Time Steps Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_all">
       <property name="text">
        <string>All frames</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton_single">
         <property name="text">
          <string>Single Frame</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_singleFrame"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="radioButton_interval">
         <property name="text">
          <string>Interval</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_intervalFirst"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_intervalLast"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_previousFrame">
         <property name="text">
          <string>Prev Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nextFrame">
         <property name="text">
          <string>Next Frame</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
